build_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num_commits=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+current_commit=$(git rev-parse --short=7 HEAD)
+version="0.1.$num_commits-$current_commit"
+opm_version="1.14.0"
+
+# Login to docker
+docker_conf="$PWD/.docker"
+mkdir -p "$docker_conf"
+docker_cmd="docker --config=$docker_conf"
```

Comment:
the boilerplate uses the `CONTAINER_ENGINE` instead of docker for those using podman

Reply:
Will use once this gets integrated with boilerplate.

Reply:
ok